usr/src/test/zfs-tests/tests/functional/zvol/zvol_swap/zvol_swap_004_pos.ksh

```diff
@@ -51,7 +51,7 @@ typeset -i min max mem
 ((min = 2 * 1024 * 1024 * 1024))
 ((max = 16 * 1024 * 1024 * 1024))
 
-for vbs in 512 1024 2048 4096 8192 16384 32768 65536 131072; do
+for vbs in 8192 16384 32768 65536 131072; do
 	for multiplier in 1 32 16384 131072; do
 		((volsize = vbs * multiplier))
```

Comment:
BTW, do the `new_volsize` checks below actually work?
In my environment adding a volume as a swap device never changes its size, volsize always stays the same.